documentation/CAMARA-OIDC-profile.md

```diff
@@ -0,0 +1,1020 @@
+
+Communication with the Backchannel Authentication Endpoint MUST utilize TLS.
+
+CIBA allows the Client to get the authentication result in three ways: poll, ping, or push. This profile only allows poll mode. In the Poll mode, the authentication result is retrieved by the Client by polling the OP's token endpoint using the new grant type.
+
+### Authentication Request
+
+An authentication request is composed of the following parameters and MAY contain additional parameters defined by extension or profile:
+
+* **scope**
+
+  REQUIRED. A space delimited and a case-sensitive list of ASCII strings for OAuth 2.0 scope values. Consistent with that, CIBA authentication requests MUST therefore contain the `openid` scope value. More information about scope values as described in Scope Parameter Section.
+
+
+* **login_hint**
+
+  REQUIRED. A hint to the OpenID Provider regarding the end-user for whom authentication is being requested. The value may contain a phone number, ip, etc., which identifies the end-user to the Authorization Server. The value may be directly collected from the user by the Client before requesting authentication at the Authorization Server, for example, but may also be obtained by other means.
+
+  The `login_hint` value will follow the format: `identifier_type:identifier_value`.
+
+  The defined values are for each identifier type:
+
+    * **_tel:+_**
+
+      For phone numbers. It must be a tel URI as defined in [RFC 3966](https://www.rfc-editor.org/info/rfc3966) for global phone numbers without visual separators in [E.164](https://www.itu.int/rec/T-REC-E.164-201011-I/en) format. For example, `tel:+34666666666`.
+
+    * **_ipport_**
+
+      For IPv4 and IPv6 addresses, that can optionally include a port. For example, `ipport:80.90.34.2:16790`, `ipport:80.90.34.2`, `ipport:[2001:db8::1]:8080` or `ipport:[2001:db8::1]`.
+
+
+Because in the CIBA flow, the Authorization Server does not have an interaction with the end-user through the consumption device, it is REQUIRED that the Client provides a `login_hint`.
+
+An authentication request is made using the HTTP POST method with the aforementioned parameters in the `application/x-www-form-urlencoded` format and a character encoding of UTF-8 in the HTTP request entity-body. When applicable, additional parameters required by the given client authentication method are also included (e.g. JWT assertion-based client authentication uses `client_assertion` and `client_assertion_type`).
+
+The client MUST authenticate with the authorization server as described in [Client Authentication Section](#client-authentication).
+
+The following is a non-normative example of an authentication request (with line wraps within values for display purposes only):
+
+```http
+POST /bc-authorize HTTP/1.1
+Host: server.example.com
+Content-Type: application/x-www-form-urlencoded
+
+scope=openid%20example-scope&
+login_hint_token=ipport%3A80.90.34.2&
```

Comment:
typo: should be ```login_hint```, not ```login_hint_token```.

Is there a consideration to use login_hint_token in the future?

Reply:
Typo fixed.

`login_hint_token` parameter could be used in scenarios with `client_secret_basic` to provide authentication, integrity and  non-repudiation; but we think that these issues can be solved in the same manner by using the `login_hint` parameter, the recommended `private_key_jwt` authentication method and TLS.

This topic has been discussed in previous conversations and several operators were in favor of using `private_key_jwt`.

Reply:
Related issue with comments supporting `private_key_jwt` https://github.com/camaraproject/IdentityAndConsentManagement/issues/104#issuecomment-1877218129.